api/cds.proto

```diff
@@ -26,17 +26,9 @@ service ClusterDiscoveryService {
   }
 }
 
-message ClusterDiscoveryRequest {
-  // The version_info provided in the request messages will be the version_info
-  // received with the most recent successfully processed response or empty on
-  // the first request.
-  bytes version_info = 1;
-  Node node = 2;
-}
-
 message ClusterDiscoveryResponse {
   bytes version_info = 1;
-  repeated Cluster clusters = 2;
+  repeated Cluster resources = 2;
```

Comment:
Why the rename? I guess it goes with resource_names, but I'm thinking in code having response->clusters() is more understandable in context than response->resources().

Throughout.